modules/sdk-core/src/bitgo/tss/ecdsa/ecdsa.ts

```diff
@@ -748,14 +748,19 @@ export async function verifyWalletSignature(params: {
   commonKeychain: string;
   userKeyId: string;
   backupKeyId: string;
-  publicShare: string;
+  decryptedShare: string;
   verifierIndex: 1 | 2;
 }): Promise<void> {
   const rawNotations = await commonVerifyWalletSignature(params);
-  const publicShareRawNotationIndex = 2 + params.verifierIndex;
+  const publicUValueRawNotationIndex = 2 + params.verifierIndex;
 
+  // Derive public form of u-value
+  const publicUValue = ecc.pointFromScalar(Buffer.from(params.decryptedShare.slice(0, 64), 'hex'), false);
+  // Verify that the u value + chaincode is equal to the proof retrieved from the raw notations
   assert(
-    params.publicShare === Buffer.from(rawNotations[publicShareRawNotationIndex].value).toString(),
+    publicUValue !== null &&
+      publicUValue.toString() + params.decryptedShare.slice(64) ===
```

Comment:
Since the chaincode is being provided in hex encoding, the publicUValue buffer needs to also be in hex encoding so please use .toString('hex') instead of .toString()
@Logicwax : context is that this fails right now while testing as the encoding for this is not correct. 
@brekiy is creating a new PR to fix this.